src/Compiler.js

```diff
@@ -31,7 +32,8 @@ async function render (htmltree, options) {
   const plugins = [
     new CurlyStylesPlugin(),
     new ScopedStylesPlugin(),
-    new InternationalizationPlugin({ translations, statistics, filters })
+    new InternationalizationPlugin({ translations, statistics, filters }),
+    new PaddingPlugin()
```

Comment:
Let's make it the first one. We'll add some other plugins later that will care about this order.